### ISSUE TITLE
Original laptop antenna or Taobao card with windows

### ISSUE THREAD

**Issue by MrJosu** [daliansky/XiaoMi-Pro#132]
Hello, i have a question. At current time i want to install windows into this laptop. During installation of taobao card i damaged pin on original laptop antenna, so i need to solder it back. For me of course best option will be to use taobao card in windows, but i think it's not working? Or exist some drivers for it?
And maybe you know how i can find original laptop antenna installed into lid?

**Comment by Menchen** [comment]
It can work under windows with driver..(try boot camp or search on Google ),  but it seem hard to use under linux...

**Comment by MrJosu** [comment]
> It can work under windows with driver..(try boot camp or search on Google ), but it seem hard to use under linux...

i think its even not possible to use bootcamp with hackintosh or i was wrong?

**Comment by FallenChromium** [comment]
Actually there is Windows drivers for this card, you may lurk into wifi issues on @stevezhengshiqi 's repository for driver link. It works under windows quite fine and works natively in most Linux distros too

**Comment by MrJosu** [comment]
> @MrJosu I've tried bootcamp but it didn't work for me since bootcamp only allowed installation on an apple device. However, I managed to get it working using the BCM43XX driver from https://www.driver-wireless.com/broadcom-bcm43xx-driver-win7810.html. Just run Install_CD > Setup.exe from the Broadcom BCM43XX Driver (Windows 10) file downloaded from this site.
> ![annotation 2019-01-05 131509](https://user-images.githubusercontent.com/38693485/50720729-663bcf00-10ed-11e9-8438-2bda92239a68.jpg)

Yes, thanks:) I managed to do same yesterday founding drivers by Hardware ID, but now another things;) I installed windows, cannot enter Mac OS, also need to found way to run bluetooth or go for a dongle, lol

**Comment by MrJosu** [comment]
@stevezhengshiqi Hey, maybe you have any idea how to make bluetooth working under windows? I mean bluetooth from taobao card

**Comment by stevezhengshiqi** [comment]
Sure. Here is BCM943602CS BT driver. I think I have uploaded it somewhere before.
[AppleBluetoothBroadcom64.zip](https://github.com/daliansky/XiaoMi-Pro/files/2792981/AppleBluetoothBroadcom64.zip)

**Comment by MrJosu** [comment]
@stevezhengshiqi Thanks:) Managed to launch, just with a bad range:( Any suggestion where i can buy spare antennas? I want to try to solder this card to lid, for better range. Because everything else in this laptop is perfectly fine and no point to move to windows only. (Plus i damaged connector when i was disconnecting original antenna)
